staging/src/k8s.io/apiserver/pkg/server/config_test.go

```diff
@@ -376,11 +376,15 @@ func TestAuthenticationAuditAnnotationsDefaultChain(t *testing.T) {
 	}
 	// these should all be the same because the handler chain mutates the event in place
 	want := map[string]string{"pandas": "are awesome", "dogs": "are okay"}
+	foundResponseComplete := false
 	for _, event := range backend.events {
+		if event.Stage == auditinternal.StageRequestReceived {
```

Comment:
Why did this change?

Reply:
`backend.events` had 2 events! looks like we had a bug before and the changes in this PR uncovered it. i set a breakpoint in this unit test and found the problem.